tests/test_ctrlc.py

```diff
@@ -5,15 +5,31 @@
 import os
 import pytest
 import signal
+import sys
 
 from aiotk import handle_ctrlc
+from unittest import mock
+
+
+if sys.platform == 'win32':
+    from ctypes import windll
+
+    GenerateConsoleCtrlEvent = windll.kernel32.GenerateConsoleCtrlEvent
+    CTRL_C_EVENT = 0
+
+    def send_ctrlc():
+        if GenerateConsoleCtrlEvent(CTRL_C_EVENT, 0) == 0:
```

Comment:
**TODO**: fix Tox to start the command using the ``CREATE_NEW_PROCESS_GROUP`` flag so CTRL-C is not sent to the Tox process.